presto-hive/src/main/java/com/facebook/presto/hive/SqlStandardAccessControl.java

```diff
@@ -160,7 +161,8 @@ public void checkCanSelectFromView(ConnectorTransactionHandle transaction, Ident
     @Override
     public void checkCanCreateViewWithSelectFromTable(ConnectorTransactionHandle transaction, Identity identity, SchemaTableName tableName)
     {
-        if (!checkTablePermission(identity, tableName, SELECT, GRANT)) {
+        if (!checkTablePermission(identity, tableName, SELECT) ||
```

Comment:
Does this go over 120 characters? If so by how many characters? If it's not too much longer than 120 I would keep it as a single line. This goes for the other IF statements you split below.